.github/workflows/pr_ping.yml

```
@@ -21,32 +21,45 @@ jobs:
       pr_title: ${{ github.event.pull_request.title }}
       pr_author: ${{ github.event.pull_request.user.login }}
       pr_repo: ${{ github.event.pull_request.base.repo.full_name }}
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
     steps:
       - name: Send notification for new PR
         if: github.event.action == 'opened'
         id: slack
-        uses: slackapi/slack-github-action@1.23.0
+        uses: slackapi/slack-github-action@1.24.0
         env:
           pr_status: ${{ github.event.pull_request.draft && ' draft ' || ' ' }}
           pr_icon: ${{ github.event.pull_request.draft && ':pr-draft:' || ':pull-request:' }}
         with:
           payload: |
             {
-              "text": "New\"$pr_status\"PR opened by \"$pr_author\" in \"$pr_repo\": #\"$pr_number\" - \"$pr_title\"",
+              "text": "New\"${{pr_status}}\"PR opened by \"${{pr_author}}\" in \"${{pr_repo}}\": #\"${{pr_number}}\" - \"${{pr_title}}\"",
```

Unfortunately these variables like `pr_status` etc that are available through `env` will not be available in this new context. Previously they worked because of shell variable interpolation. But now these are using GitHub Action variables. Luckily, there is an easy way to resolve this, using the `env` context variable: https://docs.github.com/en/actions/learn-github-actions/variables#using-contexts-to-access-variable-values

For this line, the change to use this would be:

```suggestion
              "text": "New\"${{ env.pr_status }}\"PR opened by \"${{ env.pr_author }}\" in \"${{ env.pr_repo }}\": #\"${{ env.pr_number }}\" - \"${{ env.pr_title }}\"",
```

You will need to update all the variable occurrences in these two actions to match that approach. Please keep in mind adding whitespace around the `{{ }}` as it makes things generally more readable later on :slightly_smiling_face:

---

Thank you so much for lots of guidance along the way!
made variables clearer and readable!
https://github.com/WordPress/openverse/pull/3577/commits/0cb727abdcc9c1c072c1f787cc15782fcc7130df